frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react';
+import { Button } from 'react-bootstrap';
+import { FaInfo } from 'react-icons/fa';
+import Control from 'react-leaflet-control';
+import styled from 'styled-components';
+import clsx from 'classnames';
+import * as L from 'leaflet';
+import { useEffect } from 'react';
+import { PropertyPopUpContext } from '../Map';
+import HeaderActions from './HeaderActions';
+import { InfoContent } from './InfoContent';
+import TooltipWrapper from 'components/common/TooltipWrapper';
+
+const InfoContainer = styled.div`
+  margin-right: -10px;
+  width: 341px;
+  min-height: 52px;
+  height: auto;
+  background-color: #fff;
+  position: relative;
+  border-radius: 4px;
+  box-shadow: -2px 1px 4px rgba(0, 0, 0, 0.2);
+  z-index: 1000;
+
+  &.closed {
+    width: 0px;
+    height: 0px;
+  }
+`;
+
+const InfoHeader = styled.div`
+  width: 100%;
+  height: 52px;
+  background-color: #1a5a96;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  padding-top: 16px;
+  border-top-right-radius: 4px;
+  border-top-left-radius: 4px;
+`;
+
+const InfoMain = styled.div`
+  width: 100%;
+  height: calc(100% - 80px);
+  padding-left: 10px;
+  padding: 0px 10px 5px 10px;
+`;
+
+const InfoIcon = styled(FaInfo)`
+  font-size: 30px;
+`;
+
+const InfoButton = styled(Button)`
+  width: 52px;
+  height: 52px;
+  position: absolute;
+  top: 0;
+  left: -51px;
+  background-color: #fff;
+  color: #1a5a96;
+  border-color: #1a5a96;
+  box-shadow: -2px 1px 4px rgba(0, 0, 0, 0.2);
+  &.open {
+    border-top-right-radius: 0;
+    border-bottom-right-radius: 0;
+    top: 0px;
```

Comment:
do you need this top 0 and the top 0 in the parent?

Reply:
fixed it, I only need the top 0 when it's open, it keeps the button at the top edge of the container